src/cmd_sym.ml

```diff
@@ -202,7 +206,7 @@ let cmd profiling debug unsafe optimize workers no_stop_at_failure no_values
       (fun failing (result, thread) ->
         let pc = Thread.pc thread in
         let symbols = thread.symbol_set in
-        let model = get_model ~symbols solver pc in
+        let model = get_model ~symbols solver pc print_solver_statistics in
```

Comment:
I think I would prefer that the printing is done directly in cmd rather than in get_model. If it is meant as summary statistics, it should be done once at the end.